arch/riscv64/handle_syscall.s

```diff
@@ -71,7 +71,7 @@ handle_syscall:
 	add a0, t0, x0
 
 	# call plugin handler
-	la t0, plugin_sc_handler
+	la t0, proxy_plugin_sc_handler
```

Comment:
@andronat The RISC-V arch needs a different handling. Could you revert all the modifications to the RISC-V backend? I'll take care of it later.